TravelSP/Network/TSPNetworkUrl.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  TSPNetworkUrl.swift
+//  TravelSP
+//
+//  Created by Vanessa Hurla on 13/10/2023.
+//
+
+import Foundation
+
+struct TSPNetworkUrl {
+	static let baseURL = "http://api.olhovivo.sptrans.com.br/v2.1"
+	static let loginPath = "http://api.olhovivo.sptrans.com.br/Login/Autenticar?token={token}"
```

Comment:
```suggestion
	static let loginPath = "http://api.olhovivo.sptrans.com.br/Login/Autenticar?token="
```

Remove the token field from here as it needs to be replaced with the actual token